### DISCUSSION TITLE
is ZKsync bridge portal down?

### DISCUSSION THREAD

**Discussion by Temiswift** [Cyfrin/foundry-full-course-cu#2870] [Q&A] [answered]
please i'm finding it hard to access the ZKsync portal from my system. it just keeps loading without showing up the page

**Comment by EngrPips** [comment, marked as answer]
Hello @Temiswift, Clear your browser cache and then retry or try another browser.

**Reply by EngrPips** [reply]
![Screenshot 2024-11-16 at 16 39 32](https://github.com/user-attachments/assets/d7dbb6b1-aabf-4a60-9e1d-cc25a507f6ab)

Works fine for me

**Reply by Temiswift** [reply]
Thanks @EngrPips, cleared my browser cache and it worked

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
